src/editor/contentState/tableBlockCtrl.js

```diff
@@ -202,6 +203,149 @@ const tableBlockCtrl = ContentState => {
     }
   }
 
+  // insert/remove row/column
+  ContentState.prototype.editTable = function ({ location, action, target }) {
+    const { start, end } = this.cursor
+    const block = this.getBlock(start.key)
+    if (start.key !== end.key || !/th|td/.test(block.type)) {
+      throw new Error('Cursor is not in table block, So you can not insert/edit row/column')
```

Comment:
I would guess `...block, so you...` or `...blocks. So you...` is correct.

Reply:
@fxha What's the meaning of `...block, so you... or ...blocks. So you...`?

Reply:
👍